llvm/lib/Support/SipHash.cpp

```diff
@@ -58,21 +38,15 @@
     v2 = ROTL(v2, 32);                                                         \
   } while (0)
 
-/*
```

Comment:
It might be worthwhile to keep (an edited version of) this comment? It wasn't immediately obvious to me what the value is of removing it.

Reply:
Fair, added it back with minor tweaks;  I didn't describe the template parameters because, frankly, I'm not sure I would have a very useful description beyond the obvious ;)